### ISSUE TITLE
Dolphin: updating packages resets dolphin toolbar to default

### ISSUE THREAD

**Issue by star-buck** [blue-systems/netrunner-issues-old#28]

**Comment by llelectronics** [comment]
Clearly an upstream bug. Dolphin should not do this.

**Comment by davidedmundson** [comment]
Check the version number in the rc file you're replacing.

If it's older than upstream it'll use upstreams.

**Comment by NuLogicSystems** [comment]
You mean like I did here:
`sed -e "s|17|19|" \
-i "$pkgdir/usr/share/default-settings/plasma5-profile/kxmlgui5/dolphin/dolphinui.rc"`
https://github.com/netrunner-rolling/PKGBUILD-ds9-common/blob/master/default-settings-kdeapps/PKGBUILD
This still needs to be packaged and uploaded when I get into the office, but still.

Because of where our file is shipped, it can not be updated by Dolphin itself. If it was under ~.local/share/kxmlgui5/dolphin/ this number would have been changed automatically. As it stands now we need to update this manually each time there is a major revision release.

**Comment by davidedmundson** [comment]
Can you not just set the version to "9999" in the replacement?

**Comment by llelectronics** [comment]
Ah ok. Thanks for the tip. 
Thats what we'll do then.

**Comment by llelectronics** [comment]
Should be fixed now with this commit: https://github.com/ds9-common/default-settings-kdeapps/commit/72fa5ae2b1f34ee561c905c8473208488c50fa9d

and in future Netrunner versions.

**Comment by star-buck** [comment]
@NuLogicSystems : same needs to be applied to Rolling?

**Comment by star-buck** [comment]
now does an updated package retro-fix the situation and restores our default toolbar config in dolphin again?

**Comment by llelectronics** [comment]
Yes that is what it should do. Basically it just ignored our config as it had an too old version. With the updated config it should read and use the config again.
